### ISSUE TITLE
Arty officer and radio op

### ISSUE THREAD

**Issue by GGSoupy** [DarklightGames/DarkestHour#506]
Both are unable to use the greyhound. Will this be added back into the game?
Also on a few maps we noticed the allies have a axis rifleman spawn on their team. Dog Green was one of the maps that i remember.

**Comment by AndrewTheel** [comment]
We are considering the radio operator's role and how it could be more fun to play.  The greyhound's artillery calling ability was removed, as it doesn't make much sense that it can be used to call arty when many other vehicles couldn't.  As a scout/recon car, the Greyhound and Sdkfz 234(1) could in the future be setup for a special squad type called Recon/Artillery Squad.  Where an artillery officer, crewman, and whoever would be able to spawn a special greyhound equipped for artillery spotting.

**Comment by cmbasnett** [comment]
@GGSoupy, please refrain from posting these types of questions as issues. Please use the Steam Community forums instead. :)

**Comment by handsm** [comment]
For reference, prior to DH 6.0 this functionality was implemented by editing the arty roles in a map to add bCanBeReconCrew=true, which allowed the arty officer and radioman to use armoured cars. This was done using the trick of copying and pasting and text editing RoleInfo actors. So the vehicle itself wasn't used to call arty; just that the officer & his radioman paired up and rode in the recon vehicle. It was never part of the official DH functionality, although I think it was discussed.

DH 6.0 removed the bCanBeReconCrew functionality so it stopped working. I made special 'arty' versions of the Greyhound and 234/1, which are used as modded vehicles added to some Good Guys maps. But they only work with maps using the legacy vehicle factory spawning system. In 'modern' maps the functionality that determines whether roles can enter armoured vehicles gets shifted into the DHSpawnManager actor and so can no longer be overridden in vehicle mods.

If the general feeling is that this kind of functionality is fair enough and not 'gamey', it's a quick job for me to implement it for the new spawning system. Not suggesting it happens automatically on all maps, just that it could be an option for the mapper to add to specific maps.
